mysql-test/suite/rocksdb/t/rocksdb_read_free_rpl_tables.test

```diff
@@ -0,0 +1,10 @@
+CREATE EVENT ev1 ON SCHEDULE EVERY 1 SECOND DO INSERT INTO t1 VALUES (1);
```

Comment:
I think the `CREATE EVENT` statement is confusing without a comment referencing the github issue (eg. see `mysql-test/suite/rocksdb/t/update.test`)

It'd also be good if we can simplify the test case, since I think we just need the `SET GLOBAL ...  = NULL` part right?

Reply:
No, `CREATE EVENT` is not needed here, I just kept it the same as reported in #844. Will simplify the test.